book.adoc

```diff
@@ -14,11 +14,34 @@ Antoine Sabot-Durand <antoine@sabot-durand.net>
 = Part 1 First step with CDI
 
 == Introduction to CDI
+=== Dependency injection
```

Comment:
I thought about dealing with this point in "Problems Solved by CDI", but perhaps it would be better to have a specific part DI. In that case I think it'd be better just after "What is CDI?" where the concept of DI will be introduced but not explained.
WDYT?